### ISSUE TITLE
Continued from other location - tablet issue 

### ISSUE THREAD

**Issue by RebornSerenity** [itsjavi/supereffective.gg#11]
https://www.amazon.com/gp/product/B08GHHT32V/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1

I was asked which tablet I'm using that won't allow screen rotation, and is incredibly glitchy / laggy -- this is the one that won't work, and I'm not quite sure why. 

Mine is a Samsung Galaxy A7 10.5 inch

**Comment by itsjavi** [comment]
as said in the other issue, it seems an issue in your device

maybe you have rotation locked
or the browser you are using is not handling the site correctly

**Comment by RebornSerenity** [comment]
I think it could be device, I did check the rotation lock, works on everything cept your site, your site is also the only one i've found which gives me issues like this, sadly because it is nice to use my tablet by my computer - I tried 3 browsers on my tablet, all seem to lag, and the app is the one that doesn't rotate properly it's stuck in portrait mode (not the Desktop Site) But both desktop site, and the app lag super bad on my device.  So yeah I'm not sure. I tried it on my chromebook which is 10 years old and it works just fine and my tablet is only 4 years old, so I'm really not sure why it's happening but I do wish I could fix it.
